packages/react-native/src/primitives/Carousel/__tests__/__snapshots__/Carousel.spec.tsx.snap

```diff
@@ -29,7 +29,7 @@ exports[`Carousel calls the orientation handler on orientation change 1`] = `
     removeClippedSubviews={false}
     renderItem={[Function]}
     renderToHardwareTextureAndroid={true}
-    scrollEventThrottle={50}
+    scrollEventThrottle={0.0001}
```

Comment:
how did this sneak in here?

Reply:
Not sure why exactly, but the react-native unit tests were failing on snapshots so I had to update them

Reply:
Just an fyi, we shouldn't need to update tests in this PR

Reply:
Maybe tests changed due to bumping from 71 to 73 (which is a major version change in RN land)

Reply:
That would be my assumption. RN ships with a jest preset that includes mocked values for native APIs, default value of `scrollEventThrottle` might have changed between major versions